sass/_chatbox.scss

```diff
@@ -635,8 +635,10 @@
         .box-flyout {
             background-color: var(--chat-head-color);
             box-shadow: none;
-            height: var(--fullpage-chat-height);
-            min-height: calc(var(--fullpage-chat-height) / 2);
+            // height: var(--fullpage-chat-height);
+            height: calc(var(--vh, 1vh) * 100);
```

Comment:
You need to set `--fullpage-chat-height` to `calc(var(--vh, 1vh) * 100);` in _variables.scss and then undo the edits in this file.